Yosemite/YosemiteTests/Stores/OrderStoreTests.swift

```diff
@@ -873,4 +887,8 @@ private extension OrderStoreTests {
         }
         return date
     }
+
+    func taxes() -> [Networking.OrderItemTax] {
+        return [Networking.OrderItemTax]()
```

Comment:
Nit: since this is a test in Yosemite layer, could the tax type be `OrderItemTax` or `Yosemite.OrderItemTax` which is a type alias of `Networking.OrderItemTax` in `Yosemite/Yosemite/Model/Model.swift`?

Reply:
It can't be `OrderItemTax` because that's ambiguous. Since we're importing Yosemite, Storage, and Networking, Xcode doesn't know which type it refers to.

I _think_ using the `Networking` layer type was done on purpose in all of the Stores unit tests, because it makes devs aware of the fact that Yosemite is the one that directly talks to the Networking layer. If you check the other unit tests, they all directly use the Networking models.